src/data/index.js

```diff
@@ -1592,7 +1592,7 @@ export default [
     id: 147,
     name: 'Implement Trie (Prefix Tree)',
     url: 'https://leetcode.com/problems/implement-trie-prefix-tree/',
-    pattern: 'DFS',
+    pattern: 'DFS',//Trie
```

Comment:
Nice catch - lets change this from `DFS` to `Trie` for now.

I've opened #10 as a ticket to allow for multiple patterns per question